watertap/unit_models/nanofiltration_DSPMDE_0D.py

```diff
@@ -2023,13 +2162,65 @@ def calculate_scaling_factors(self):
             sf = iscale.get_scaling_factor(self.flux_mol_phase_comp[t, x, p, j])
             iscale.constraint_scaling_transform(con, sf)
 
-        for con in self.eq_electroneutrality_pore.values():
-            iscale.constraint_scaling_transform(con, 1e-2)
-        for con in self.eq_electroneutrality_permeate.values():
-            iscale.constraint_scaling_transform(con, 1e-2)
+        for (t, x, y, p), con in self.eq_electroneutrality_pore.items():
+            sf = (
+                sum(
+                    (
+                        iscale.get_scaling_factor(
+                            self.pore_entrance[t, x].conc_mol_phase_comp[p, j]
+                        )
+                        * iscale.get_scaling_factor(
+                            self.pore_entrance[t, x].charge_comp[j]
+                        )
+                    )
+                    ** -1
+                    for j in self.config.property_package.solute_set
+                )
+                ** -1
+            ) ** -1
```

Comment:
Hmm... this is all well and good, following that logic of equivalent resistance in parallel circuits, but do you have an extra power of -1 that should be removed here?

Reply:
Its becouse we need to sum this with scailing factor for membrane charge which I do later down. 
 `(e.g ((sum(x^-1)^-1)^-1+mem_charge^-1)^-1`

Reply:
Makes sense- we could then cancel these powers out, but won't functionally change anything. I'll leave the choice to you.